### PR TITLE
Awesome Books ES6 modules

### PR DESCRIPTION
**Pull request by DeVuDeveloper**
Hi, Project is set with ES6 . Directory Modules were added and code refactored. Luxon added. 
Everything is working fine, page, no linters errors. The big problem was linter error prefer const instead of let. I put class Book in module and  I exported it together with let(array), but linter prefers const what is not possible in this approach. So I exported what was possible. 
In-Browser all work perfectly, time and everything in GitHub live preview, luxon shows error 404. 
what prevent the page to works properly.
Thank you.
Update:
Required changes made. Functions refactored in ES6 syntax.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const date = document.getElementById('date');
+date.innerText = DateTime.now().toFormat('MMMM dd, yyyy, hh:mm:ss a').toString();
+
+let books = [];
+
+function getStore(localBooks) {
+  if (localBooks) {
+    books = JSON.parse(localBooks);
+  }
+}
```
Comment (Lameck1) [inline index.js]: - [ ] Good job implementing the functionality. However, we are expected to strictly adhere to ES6 principles. Please let's refactor all functions to ES6 format. The requirement for this is quoted below: 

  > - Use ES6 syntax.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  setStore(books);
+  titleInput.value = '';
+  authorInput.value = '';
+  renderBook(renderBooks, books);
+}
+function deleteBook(e) {
+  Book.deleteBook(e.target.rel);
+  setStore(books);
+  renderBook(renderBooks, books);
+}
```
Comment (Lameck1) [inline index.js]: - [ ] Same issue as above for these two functions. Please refactor as necessary.

File: modules/render.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    const remove = document.createElement('li');
+    const header = document.createElement('h2');
+    renderBooks.appendChild(oneBook);
+    oneBook.appendChild(titleByAuthor);
+    titleByAuthor.appendChild(header);
+    header.innerText = `"${book.title}" by ${book.author}`;
+    oneBook.appendChild(remove);
+    remove.innerHTML = `<a class='delete' rel='${book.id}' href="#">Remove</a>`;
+  });
+}
```
Comment (Lameck1) [inline modules/render.js]: - [ ] Just as above, please let's do away with the function keyword and refactor this function to use ES6 syntax.

File: modules/setStore.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export default function setStore(books) {
+  localStorage.setItem('booksStore', JSON.stringify(books));
+}
```
Comment (Lameck1) [inline modules/setStore.js]: - [ ] This one too, same issue as above.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @VuDej,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @VuDej,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**